source/fundamentals/builders/indexes.txt

```diff
@@ -11,3 +11,150 @@ Indexes Builders
    :class: singlecol
 
 .. _indexes-builders:
+
+Overview 
+--------
+
+This guide shows you how to specify **indexes** using the Java Driver
+:doc:`builders </fundamentals/builders/>`. Index builders provide
+convenient methods for different types of indexes:
+
+- :ref:`Ascending <index-ascending>`
+- :ref:`Descending <index-descending>`
+- :ref:`Compound Indexes <index-compound-indexes>`
+- :ref:`Text Indexes <index-text-indexes>`
+- :ref:`Hashed Indexes <index-hashed-indexes>`
+- :ref:`Geospatial Indexes <geospatial-indexes>`
+
+Like an index in a textbook, an index in MongoDB groups data based
```

Comment:
I like that you're using using an analogy, but I think this one might need more explanation. I should be able to read the description of how MongoDB groups data in the context of a textbook which would read:

In an index in a textbook, data is grouped based on the field you provide and an index builder.

Not sure what that corresponds to in a textbook. Maybe you can just limit the sentence to what parts of it are exactly the same and then describe the details specific to MongoDB in a following sentence.